webapp/packages/core-connections/src/Administration/Connections/ConnectionForm/Options/OptionsController.ts

```diff
@@ -107,33 +111,56 @@ implements IInitializableController {
     this.updateName();
   }
 
+  private getParametersFromUrl(url: string) {
+    const parameters: IParsedUrlParameters = {};
+    const isParameterValidRegex = /[\[,\],{,}]+/;
```

Comment:
`[\[\]{},]+` ? but you sure about `,` character? seems it's not a part of special chars